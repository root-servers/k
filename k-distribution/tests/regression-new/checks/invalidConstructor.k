module INVALIDCONSTRUCTOR
  imports INT
  syntax K ::= foo()
  syntax Int ::= bar()
endmodule
