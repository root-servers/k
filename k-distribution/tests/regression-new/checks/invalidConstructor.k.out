[Error] Compiler: Cannot add new constructors to hooked sort K
	Source(invalidConstructor.k)
	Location(3,16,3,20)
[Error] Compiler: Cannot add new constructors to hooked sort Int
	Source(invalidConstructor.k)
	Location(4,18,4,22)
[Error] Compiler: Had 2 structural errors.
